README.md

```diff
@@ -125,15 +125,14 @@ stacks:
 ## Directories
 
 - `templates` - CloudFormation, SparkleFormation or CfnDsl templates.
-- `polices` - Stack policies.
```

Comment:
Not only is `polices` a typo, but looks to be duplicated by the last line in this list.